webcomponents/tool/src/main/frontend/sakai-editor.js

```diff
@@ -0,0 +1,42 @@
+import {SakaiElement} from "./sakai-element.js";
+import {html} from "./assets/lit-element/lit-element.js";
+
+class SakaiEditor extends SakaiElement {
+    static get properties() {
+        return {
+            textAreaId: { type: String },
```

Comment:
You need to make these Edge friendly, so textAreaId: { attribute: text-area-id, type: String }

But, what about when the source html isn't coming from a text area? You can get markup from divs, too. I'd use sourceMarkupId: { attribute: "source-markup-id", type: String }